package.json

```diff
@@ -98,8 +97,9 @@
     "npmpub": "^3.1.0",
     "object-assign": "^3.0.0",
```

Comment:
Since PostCSS 6 requires `node >= 4` anyways, maybe time to get rid off that if it doesn't require to much changes 😛